### ISSUE TITLE
Problem with meaningful sequence on product tile

### ISSUE THREAD

**Issue by alisonrae** [PublicisSapient/accessible-ecommerce-demo#40]
Problem with current flow in the product tile.

The problem is that the number of products is defaulted to 1, but I haven't decided to buy that product yet. As I tab through, it sounds like a quantity of 1 has been selected, but I did not decide to buy that yet.

A better option would be that I add the item to the cart and THEN have the option to increase the quantity (or remove it). See an example on Loblaws click and collect (sort of works).

Make the flow go like this:

Tab stop 1: Product name + price (same as current)
Tab stop 2: Add to cart button. [Aria live attribute says, "One item [item name] has been added to cart"]
--- If i add an item to my cart, that reveals the change quantity option BELOW add to cart:
Tab stop 3: Decrease quantity button, with text label of "Remove [item name] from cart" if there is only one item in the cart (so by default). If there is more than one item in the field, the label will say "Decrease quantity [item name]"
Tab stop 4: Text field with label "Quantity"
Tab stop 5: Increase quantity button that announces the label and the current quantity. Label should be "Increase Quantity [item name]".

Need UX team to give this some thought and update it. [DONE -- now assigned to a developer]

**Comment by alisonrae** [comment]
Fixed
